### PR TITLE
Cqw 30 updated version of join us

### PR DESCRIPTION
**Pull request by aberracja**
* [x] update 'join us' page
* [x] fix spec
* [x] remove ugly link borders and highlights

Reviewers:
* [ ] @MagdaMalinowska 
* [x] @rpietraszko 
* [x] @AdamMankowski 
* [ ] @beatadelura 
* [x] @pjar 
* [ ] @kruszczynski

### CODE REVIEW

**Comment by AdamMankowski** [comment]
There needs to be a bottom margin after email. Please check all resolution before you push your changes. Its on tablet and mobile view
code_quest_is_a_ruby_on_rails_and_ios_development_studio_

File: source/partials/_svg_upper_part.html.slim [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-    = partial 'svg/join_our_crew/wave_1'
-  .svg.wave-3
-    = partial 'svg/join_our_crew/wave_3'
-  .svg.wave-4.hidden-on-mobile
-    = partial 'svg/join_our_crew/wave_1'
-  .svg.wave-5.hidden-on-mobile
-    = partial 'svg/join_our_crew/wave_1'
-  .svg.wave-6.hidden-on-mobile
-    = partial 'svg/join_our_crew/wave_3'
-  .svg.cloud-diver.hidden-on-mobile
```
Comment (AdamMankowski) [inline source/partials/_svg_upper_part.html.slim]: Are we using those svg somewhere else? If not i think we should get rid of them

Comment (beatadelura) [inline source/partials/_svg_upper_part.html.slim]: ..and also styles should be removed for those svgs.

Comment (aberracja) [inline source/partials/_svg_upper_part.html.slim]: done

File: source/stylesheets/modules/_m-join-us.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
+.join-our-crew {
+  .m-footer { margin-top: 20px; }
```
Comment (rpietraszko) [inline source/stylesheets/modules/_m-join-us.scss]: @aberracja FYI We agreed to not use one-liners in CQW as far as I remember

Comment (aberracja) [inline source/stylesheets/modules/_m-join-us.scss]: good to know!

**Pull request merged**